vars/main.yml

```diff
@@ -1,2 +1,4 @@
 ---
 # vars file for supervisor
+
+
```

Comment:
I'm not aware of that standard. But if it really is, no problem from me. I just wanted to know why there were spaces.